src/listener/server-profiles.js

```diff
@@ -162,7 +167,12 @@ export default class ServerProfilesListener extends Listener {
 
     dialog.addChoice(/yes/i, () => {
       this.client.ServerProfiles.getServerProfile(msg.profileId).then((res) => {
-        return this.serverHardware.PowerOffHardware(res.serverHardwareUri, msg);
+        if (res.serverHardwareUri === null) {
+          return this.transform.text(msg, msg.message.user.name + ", " + this.transform.hyperlink(profileHyperlink, profileName) + " does not have any assigned server hardware to power off. Try assigning server hardware to the profile or try powering off a different profile.");
```

Comment:
I would remove this from the last sentence "or try powering off a different profile."